django_cloud_tasks/tasks/task.py

```diff
@@ -182,9 +182,20 @@ def sync(cls, **kwargs):
         return cls().run(**kwargs)
 
     @classmethod
-    def asap(cls, **kwargs):
-        return cls.push(task_kwargs=kwargs)
-
+    def asap(
+        cls,
+        headers: dict | None = None,
+        queue: str | None = None,
+        delay_in_seconds: int | None = None,
```

Comment:
The method asap should not suport delay in seconds. We have another method called later to do it